src/app/reducers/vm/redux/vm.effects.ts

```diff
@@ -394,6 +390,7 @@ export class VirtualMachinesEffects {
                 id: notificationId,
                 message: 'JOB_NOTIFICATIONS.VM.DESTROY_FAILED'
               });
+              this.update(action.payload, VmState.Error);
```

Comment:
I think we should pass a VM and an error to vmUpdateError to get rid of this.update calls